riptide-core/src/main/java/org/zalando/riptide/tryit/TryWith.java

```diff
@@ -0,0 +1,91 @@
+package org.zalando.riptide.tryit;
```

Comment:
Do you think it should stay in the main directory? I thought it might clutter the public visible API. In general I have the feeling, that the concept of Throwing* and Try* is more generic concept that riptide should import it from another project.